packages/go_router_builder/example/lib/main.dart

```diff
@@ -232,7 +232,7 @@ class PersonScreen extends StatelessWidget {
                     person.id,
                     entry.key,
                     $extra: ++_extraClickCount,
-                  ).go(context),
+                  ).go(context, extra: _extraClickCount),
```

Comment:
why does it need to pass _extraClickCount to extra if it is already assigned to $extra?

Reply:
I can remove assigning value to $extra